src/common/pubsub.ts

```diff
@@ -124,6 +126,30 @@ export const notifyCommentFeatured = async (
     commentId,
   });
 
+export const notifyUsernameChanged = async (
```

Comment:
I'm not sure if it's better to publish this message here or from the gateway.
Because originally the gateway is in charge of the user entity.
What do you think?

Reply:
Yes, was actually thinking of that initially, but I've thought of ensuring first the entity has changed already, so I made it watch from the API's DB. What do you think?

Reply:
We can do the same in the gateway, look for the changes there. I think it's better in the current architecture